fenix/app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -563,6 +568,8 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
                 "finishing" to isFinishing.toString(),
             ),
         )
+
+        navigationBinding.stop()
```

Comment:
Whoops, this isn't actually necessary 😅 handled automatically by adding the binding to the list of lifecycle observers above.